src/foam/u2/layout/Card.js

```diff
@@ -0,0 +1,140 @@
+/**
+ * @license
+ * Copyright 2019 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.u2.layout',
+  name: 'Card',
+  extends: 'foam.u2.borders.CardBorder',
+  documentation: `
+    A card based on a responsive grid system
+  `,
+
+  imports: [
+    'displayWidth?'
+  ],
+
+  css: `
+    ^ {
+      margin: 8px;
+    }
+  `,
+
+  properties: [
+    {
+      class: 'Int',
+      name: 'cols',
+      documentation: `
+        Sets up a standard default column width across all display types
+      `,
+      value: 12
+    },
+    {
+      class: 'Int',
+      name: 'xxsCols',
+      documentation: `
+        The column width for the smaller end of smartphone devices and smartphone portrait screens:
```

Comment:
I think the majority of the documentation on these properties are better off on the enum values. I wouldn't call out pixel values in this model. It only cares about DisplayWidths.

Reply:
Done.